third_party/zlib/BUILD.gn

```diff
@@ -30,10 +30,12 @@ static_library("zlib_x86_simd") {
 
 config("zlib_warnings") {
   visibility = [ ":*" ]
-  cflags = [
-    "-Wno-incompatible-pointer-types",
-    "-Wno-shift-negative-value",
-  ]
+  if (!is_win) {
```

Comment:
Now that I think about it. It may be due to differences in host vs target builds. I gated it on `is_clang` as suggested.